repository/src/main/java/org/alfresco/repo/security/person/SplitPersonCleanupBootstrapBean.java

```diff
@@ -1,32 +1,33 @@
-/*
- * #%L
- * Alfresco Repository
- * %%
- * Copyright (C) 2005 - 2016 Alfresco Software Limited
```

Comment:
Please change the year from 2016 to 2024.

Reply:
Done